### PR TITLE
feat: added new constraint rules and fixed error in TextListCell

### PR DESCRIPTION
**Pull request by renanmassaroto**
## Description

Added new constraint rules in OceanConstraintsDSL

Fixes # (issue)

Fixed a bug introduced in TextListCell in [PR](https://github.com/ocean-ds/ocean-ios/pull/229)

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

## Screenshots (if appropriate):

https://user-images.githubusercontent.com/114941235/219162868-d6d01396-3c20-4e01-92ca-cb4721d6f7b2.mov

## Checklist:

- [X] I have read the [CONTRIBUTING](CONTRIBUTING.md) document
- [X] I have performed a self-review of my own code
- [X] My changes generate no new warnings
- [X] I have made corresponding changes to the documentation (if appropriate)
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] All new and existing tests passed

### CODE REVIEW

File: OceanComponents/Classes/Components/TextList/Ocean+TextListCell.swift [hunk 1]
```diff
@@ -270,10 +276,14 @@ extension Ocean {
...
 
-        private lazy var textLabel: UILabel = {
-            Ocean.Typography.caption { label in
-                label.numberOfLines = 1
-            }
+        private lazy var textLabel: TextLabel = {
+            let label = TextLabel()
+            label.font = .baseRegular(size: Ocean.font.fontSizeXxxs)
+            label.boldSize = Ocean.font.fontSizeXxxs
+            label.textColor = Ocean.color.colorInterfaceDarkDown
```
Comment (AcassioM) [inline OceanComponents/Classes/Components/TextList/Ocean+TextListCell.swift:283]: vai setar a cor verde no projeto blu?

File: OceanDesignSystem/Controllers/Components/TextListViewController.swift [hunk 2]
```diff
@@ -141,9 +141,19 @@ final public class TextListViewController : UIViewController {
                                                imageIcon: Ocean.icon.exclamationCircleSolid,
                                                color: Ocean.color.colorStatusNeutralDeep)
         }
+        
+        let textList18 = Ocean.TextList.cell { textList in
+            textList.title = "Title"
+            textList.textTextLabel = Ocean.TextLabelModel(value: "Text", bold: true, color: Ocean.color.colorStatusPositiveDeep)
```
Comment (AcassioM) [inline OceanDesignSystem/Controllers/Components/TextListViewController.swift:147]: pergunta respondida... rsss

File: OceanDesignSystem/Controllers/Components/TextListViewController.swift [hunk 3]
```diff
@@ -175,10 +188,14 @@ final public class TextListViewController : UIViewController {
     }
 
     private func applyScrollViewDefaultContraints() {
-        scrollView.setConstraints((.fillSuperView, toView: view))
-        scrollableContentView.setConstraints(([.fillSuperView,
-                                               .centerHorizontally], toView: scrollView),
-                                             ([.fullWidth], toView: view))
+        scrollView.oceanConstraints
```
Comment (AcassioM) [inline OceanDesignSystem/Controllers/Components/TextListViewController.swift:191]: lindão

**Review by AcassioM — APPROVED** [review]

File: OceanComponents/Classes/Components/TextList/Ocean+TextListCell.swift (hunk 1 above)
Comment (renanmassaroto) [reply OceanComponents/Classes/Components/TextList/Ocean+TextListCell.swift:283]: Isso e só onde for necessário! :D

**Review by vromeiro — APPROVED** [review]

**Pull request merged**